tests/Wallabag/CoreBundle/Controller/TagControllerTest.php

```diff
@@ -126,9 +126,13 @@ public function testRemoveTagFromEntry()
             ->getRepository('WallabagCoreBundle:Tag')
             ->findOneByEntryAndTagLabel($entry, $this->tagName);
 
+        // We make a first request to set an history and test redirection after tag deletion
+        $client->request('GET', '/view/'.$entry->getId());
```

Comment:
@j0k3r @nicosomb this new request is making the next `assertNotContains` call to fail.

It seems that `$entry` is not correctly updated when accessed during assertion. Do you have any idea?